components/sync/engine_impl/brave_model_type_worker.cc

```diff
@@ -12,6 +12,18 @@
 
 namespace syncer {
 
+namespace {
+// Between each failed commit the timeout is randomly increased,
+// see |BackoffDelayProvider|.
+// 7 attemps gives near 2..5 minutes before resetting progress marker
+// and new get updates
+
+size_t kFailuresToResetMarker = 7;
+// Allow re-enable types not often than once in 30 minutes
```

Comment:
not "re-enable types" anymore

Reply:
Fixed at https://github.com/brave/brave-core/pull/8201/commits/b2f3c2f17ff4dabba0f4761c4b1830f98a9dafbc